mushroomsoft/app/src/app/components/footer/footer.component.ts

```diff
@@ -0,0 +1,22 @@
+import {Component} from '@angular/core';
+import locationItems from '../../../config/location.json';
+
+@Component({
+  selector: 'app-footer',
+  templateUrl: './footer.component.html',
+  styleUrls: ['./footer.component.scss']
+})
+export class FooterComponent {
+  public titleLocation = locationItems.title.title;
+  public descriptionLocation = locationItems.title.description;
+  public imageLocation = locationItems.title.img;
+  public titleAddress = locationItems.location.title;
+  public streetAddress = locationItems.location.address;
+  public locationAddress = locationItems.location.location;
+  public floorAddress = locationItems.location.floor;
+  public phoneAddress = locationItems.location.phone;
+  public socialImage = locationItems.footer.social;
+  public socialClassImage = locationItems.footer.classSocial;
+  public logoFooterImage = locationItems.footer.logo;
+  public logoClassImage = locationItems.footer.classLogo;
```

Comment:
what do you think if we create an interface or object instead of creating it one by one?